src/shared/containers/Refile/RefileErrorsForm.jsx

```diff
@@ -151,11 +218,28 @@ class RefileErrorsForm extends Component {
           startDate,
           endDate,
           offset,
+          resultLimit,
         }
       ).then(response => {
-        console.log('Form Successful Response: ', response);
         this.props.setApplicationLoadingState(false);
-        this.setState({...this.baseState, formResult: { processed: true } });
+
+        this.setState({
+          ...this.baseState,
+          formResult: { processed: true },
+          refileErrorResultsTotal: response.data.data.totalCount,
+          refileErrorResults: response.data.data.data,
```

Comment:
`data.data.data`!!! More data!

[I know we can't directly affect this]

Reply:
No... I don't like it either lol